<launch>
  <group ns="axis">
    <node pkg="axis_camera" type="teleop_twist.py" name="teleop">
        <param name="enable_button" value="1"/>
        <param name="scale_pan_deg" value="25"/>
        <param name="scale_tilt_deg" value="25"/>
    </node>
  </group>
</launch>
